src/components/App.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/jsx-no-bind */
+import React, { useState, useEffect } from "react";
+import "../styles/App.css";
+import getForecast from "../requests/getForecast";
+import LocationDetails from "./LocationDetails";
+import ForecastSummaries from "./ForecastSummaries";
+import ForecastDetails from "./ForecastDetails";
+import SearchForm from "./SearchForm";
+
+function App() {
+  const [forecasts, setForecasts] = useState([]);
+  const [location, setLocation] = useState({
+    city: "Hi",
```

Comment:
Here I would initialise the city property with an empty string "" rather than the string of "Hi". This is so that if the user accidentally sees this state, it would look confusing and unprofessional